R/init.R

```diff
@@ -1,7 +1,23 @@
-if (interactive() &&
-  Sys.getenv("RSTUDIO") == "" &&
-  Sys.getenv("TERM_PROGRAM") == "vscode") local({
 
+# This file is executed with its containing directory as wd
+
+# Remember absolute path of vsc.R
+dir_init <- getwd()
+
+
+# This function is run at the beginning of R's startup sequence
+# Code that is meant to be run at the end of the startup should go in `initLast`
+initFirst <- function() {
```

Comment:
I guess we should stick to our existing snake_case naming convention in all newly introduced symbols?

Reply:
Thanks for fixing the script paths. Feel free to adjust naming/casing as you see fit. Considering the chaos of naming conventions in R itself, I have kind of given up on this topic.. :D